README.md

```diff
@@ -53,7 +53,7 @@ resources:
 * [#node.js channel on chat.freenode.net][]
 * [Node.js Discord Community](https://discordapp.com/invite/v7rrPdE)
 * [Node.js Slack Community](https://node-js.slack.com/): Visit
-  [nodeslackers.com](http://www.nodeslackers.com/) to register.
+  [nodeslackers.com](http://www.nodeslackers.com/) to register
```

Comment:
Maybe something like this instead?

```markdown
* [Node.js Slack Community](https://node-js.slack.com/)
  * To register: [nodeslackers.com](http://www.nodeslackers.com/)
```

Reply:
Thank you for your suggestion, I will try it.

Reply:
Sorry to reply to your comment so late, you know that it is 8:00 in the morning, I just got up.